samples/stock/demo/all-indicators/demo.html

```diff
@@ -0,0 +1,65 @@
+<script src="https://code.highcharts.com/stock/highstock.js"></script>
+<script src="https://code.highcharts.com/stock/modules/data.js"></script>
+<script src="https://code.highcharts.com/stock/modules/exporting.js"></script>
+<script src="https://code.highcharts.com/stock/indicators/indicators-all.js"></script>
+
+<div class='main'>
+  <div class='selectors'>
+    <div class="left">
+      <label class='left__labels' for="overlays">Overlays:</label>
+      <select class='left__select' id="overlays">
+        <option value="none">None</option>
```

Comment:
I haven't seen many HTML tags in our official demos. We could consider using Highcharts tools e.g. `H.createElement` and then we could position 'selects' using Highcharts coordinates (without CSS). @pawelfus what do you think?

Reply:
I think simple HTML in the demo is fine. For example Periodic table: http://utils.highcharts.local/samples/#view/highcharts/series-heatmap/periodic-table